src/OpenSpaceToolkit/Physics/Coordinate/Axes.cpp

```diff
@@ -105,7 +105,7 @@ Axes Axes::inFrame(const Shared<const Frame>& aFrameSPtr, const Instant& anInsta
 {
     using ostk::physics::coordinate::Transform;
 
-    if ((aFrameSPtr == nullptr) && (!aFrameSPtr->isDefined()))
+    if ((aFrameSPtr == nullptr) || (!aFrameSPtr->isDefined()))
```

Comment:
Nice catch! Is this the only occurence?

Reply:
It's the only one that the compiler complained about, so likely an isolated issue!